VimeoNetworking/Sources/AuthenticationController.swift

```diff
@@ -82,13 +82,13 @@ final public class AuthenticationController
      
      - returns: a new `AuthenticationController`
      */
-    public init(client: VimeoClient)
+    public init(client: VimeoClient, appConfiguration: AppConfiguration)
     {
-        self.configuration = client.configuration
+        self.configuration = appConfiguration
         self.client = client
-        self.accountStore = AccountStore(configuration: client.configuration)
+        self.accountStore = AccountStore(configuration: configuration)
         
-        self.authenticatorClient = VimeoClient(appConfiguration: client.configuration)
+        self.authenticatorClient = VimeoClient(appConfiguration: configuration)
```

Comment:
@mikew-personal @fyell here's the answer         
/// We need to use a separate client to make the actual auth requests, to ensure it's unauthenticated
    private let authenticatorClient: VimeoClient